### PR TITLE
OSR initial page

### PR DESCRIPTION
**Pull request by andrewfraser73**
This is the first draft of the OSR (Operations Status Report) none of the drill downs are in place.  Also the initial report contains other data around the run times and ATE and board pass rates

Have also written a CSV export

The main UI is an amalgamation of tabs and table.

### CODE REVIEW

File: src/Facade/ResourceBuilders/ProductionMeasuresResourceBuilder.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public ProductionMeasuresResource Build(ProductionMeasures productionMeasures)
                 StockValue = productionMeasures.StockValue,
                 BackOrderValue = productionMeasures.BackOrderValue,
                 PboJobId = productionMeasures.PboJobId,
-                OldestBackOrder = productionMeasures.OldestBackOrder?.ToString("o"),
+                OldestBackOrder = productionMeasures.OldestBackOrder?.ToString("dd-MMM-yy"),
```
Comment (atcampbell) [inline src/Facade/ResourceBuilders/ProductionMeasuresResourceBuilder.cs]: Why did you need to change this from an iso string?

File: src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+    const shortagesBaseHref = `${config.appRoot}/production/reports/measures/shortages`;
+    const backOrdersBaseHref = `${config.appRoot}/production/reports/measures/back-orders`;
+    const builtThisWeekBaseHref = `${config.appRoot}/production/reports/measures/built-this-week`;
+    const fflagStockBaseHref = `${config.appRoot}/production/reports/stock/f-flag`; // may already exist in linnapps
+
+    function handleChange(event, newValue) {
+        setValue(newValue);
+    }
+
+    function round(num) {
```
Comment (atcampbell) [inline src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js]: Is it worth adding this to the shared library?

File: src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+                        <TabCell index={0} value={tabValue} />
+                        <TabCell index={0} value={tabValue} />
+                        <TabCell index={0} value={tabValue} />
+                        <TabCell index={0} value={tabValue} align="centre" colSpan={2}>
+                            Days
+                        </TabCell>
+                        <TabCell index={0} value={tabValue} align="centre" colSpan={2}>
+                            Can Do
+                        </TabCell>
+                        <TabCell index={0} value={tabValue} />
```
Comment (atcampbell) [inline src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js]: What are these value={tabValue} doing?

File: src/Facade/ResourceBuilders/ProductionMeasuresResourceBuilder.cs (hunk 1 above)
Comment (andrewfraser73) [reply src/Facade/ResourceBuilders/ProductionMeasuresResourceBuilder.cs]: it was what was displayed on the OSR report converting it to a string with full ISO pattern would then require further work downstream when this is literally the only place it will be consumed

File: src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js (hunk 2 above)
Comment (andrewfraser73) [reply src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js]: maybe certainly if it is used on another component there is a case for moving it

File: src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js (hunk 3 above)
Comment (andrewfraser73) [reply src/Service.Host/client/src/components/reports/measures/ProductionMeasuresCitTable.js]: maybe value isn't the best description it is the index of the currently selected tab hence tabValue.  i will ponder on whether there is a better name for this.
